strawberry/arguments.py

```diff
@@ -119,15 +134,15 @@ def get_arguments_from_annotations(
             argument = StrawberryArgument.from_annotated(
                 python_name=name,
                 annotation=annotation,
-                default_value=default_value,
+                default=default_value,
                 origin=origin,
             )
         else:
             argument = StrawberryArgument(
                 type_=annotation,
                 python_name=name,
                 graphql_name=None,
-                default_value=default_value,
+                default=default_value,
```

Comment:
Can we rename the `default_value`s to `default` as well?

Reply:
Yep will do